File: Python to Dalvik Compiler/CodigosTeste/micro04/micro04.py
def micro04():
	x,num,intervalo = 0,0,0

	for x in range(5):
		print("Digite o numero: ",end="")
		num = int(input())
		if num >=10:
			if num <=150:
				intervalo = intervalo +1

	print("Ao total, foram digitados "+str(intervalo)+" numeros no intervalo entre 10 e 150")

micro04()

File: Python to Dalvik Compiler/CodigosTeste/micro04/micro04.java
import java.util.Scanner;

public class micro04
{
	public static void main(String[] args)
	{
		Scanner s = new Scanner(System.in);
		int x=0,num=0,intervalo =0;

		for (x=0;x<5;x++){
			System.out.print("Digite o numero: ");
			num = s.nextInt();
			if( num >=10)
				if (num <=150)
					intervalo = intervalo +1;
		}
		
		System.out.println("Ao total, foram digitados "+intervalo+" numeros no intervalo entre 10 e 150");
	}
}